boards/native/Makefile.dep

```diff
@@ -19,3 +19,9 @@ ifneq (,$(filter socket_zep,$(USEMODULE)))
   USEMODULE += checksum
   USEMODULE += random
 endif
+
+ifneq (,$(filter netdev_default gnrc_netdev_default,$(USEMODULE)))
+  USEMODULE += iolist
+  USEMODULE += periph_hwrng
```

Comment:
These (both `iolist` and `periph_hwrng`) should rather be a dependency of `netdev_raw802154`. (Also, but maybe @cladmi can help you better with this: I think `periph_hwrng` should rather be pulled in with `FEATURES_REQUIRED`)